src/components/Viewport.ts

```diff
@@ -844,7 +850,7 @@ export default class Viewport {
     // Update panel position && fit to wrapper
     let nextPanelPos = firstPanel.getPosition();
     let maintainingPanel: Panel = firstPanel;
-    if ((currentState.holding || currentState.playing) && nearestPanel) {
+    if (nearestPanel) {
```

Comment:
@WoodNeck 

What kind of conditions is this?

Reply:
This was intended to update panel positions correctly only when in holding/animating state.
But in free scroll mode, it was also needed that it should be updated same in the idle state.